skycoin/skywire
# Rebuild Hypervisor UI

Did you run `make format && make check`? Yes

Fixes #_

 Changes:	
- update UI for new changes (remove reboot bottom, added shutdown bottom, ...)

How to test this PR:
- build
- run visor as hypervisor with UI
- check UI changes
